corehq/apps/accounting/bootstrap/utils.py

```diff
@@ -117,38 +66,40 @@ def _ensure_product_rate(monthly_fee, edition, verbose, apps):
 
     SoftwareProductRate = apps.get_model('accounting', 'SoftwareProductRate')
 
-    product_name = 'CommCare %s' % edition
+    product_name = f"CommCare {edition}"
     if edition == SoftwarePlanEdition.ENTERPRISE:
-        product_name = "Dimagi Only %s" % product_name
+        product_name = f"Dimagi Only {product_name}"
 
     product_rate = SoftwareProductRate(monthly_fee=monthly_fee)
     try:
-        # TODO - remove after squashing migrations
-        SoftwareProduct = apps.get_model('accounting', 'SoftwareProduct')
-        product = SoftwareProduct(name=product_name, product_type='CommCare')
-        try:
-            product = SoftwareProduct.objects.get(name=product.name)
-            if verbose:
-                log_accounting_info(
-                    "Product '%s' already exists. Using existing product to add rate."
-                    % product.name
-                )
-        except SoftwareProduct.DoesNotExist:
-            if verbose:
-                log_accounting_info("Creating Product: %s" % product)
-            product.save()
+        product = _get_software_product(product_name, verbose, apps)
         product_rate.product = product
+    except LookupError:
+        product = None
+        product_rate.name = product_name
 
-        if verbose:
-            log_accounting_info("Corresponding product rate of $%d created." % product_rate.monthly_fee)
+    product_rate.save()
+    if verbose:
+        log_accounting_info(f"Corresponding product rate of {product_rate.monthly_fee} created.")
 
-        return product, product_rate
+    return product, product_rate
 
-    except LookupError:
-        product_rate.name = product_name
+
+def _get_software_product(product_name, verbose, apps):
+    # SoftwareProduct no longer exists but is retained here to avoid breaking old migrations
```

Comment:
I fully agree, but I'm not actually sure how to do that. `SoftwareProduct` is currently created in accounting migration `0001_squashed_0052_ensure_report_builder_plans`, modified in `0010_remove_softwareproduct_product_type` and deleted entirely in `0011_remove_softwareproduct`, all of which are ~7 years old. The similarly aged comment that stood here previously noted that this could be removed "after squashing migrations" -- I'm not sure if there was more squashing intended or if it was just not fully correct.